nmap-update/Makefile.in

```diff
@@ -59,6 +59,6 @@ Makefile: Makefile.in
 
 makefile.dep:
 	$(CC) -MM $(CPPFLAGS) $(C_FILES) > $@
-include makefile.dep
+	include makefile.dep
```

Comment:
I don't think this is the right fix for the errors you said you were getting. Can you be more specific about what the errors are and when they show up?